mne/_fiff/meas_info.py

```diff
@@ -1935,15 +1935,24 @@ def _repr_html_(self):
         info_template = _get_html_template("repr", "info.html.jinja")
         return info_template.render(info=self)
 
-    def save(self, fname):
+    @verbose
+    def save(self, fname, overwrite=False, verbose=None):
```

Comment:
Good to add kw-only when we can!
```suggestion
    def save(self, fname, *, overwrite=False, verbose=None):
```